worker/uniter/charm/bundles.go

```diff
@@ -71,11 +70,12 @@ func (d *BundlesDir) download(info BundleInfo, target string, abort <-chan struc
 	expectedSha256, err := info.ArchiveSha256()
 	req := downloader.Request{
 		URL:       curl,
-		TargetDir: d.downloadsPath(),
+		TargetDir: downloadsPath(d.path), // XXX check this
```

Comment:
To remind me to check that this was definitely correct when QAing (which I did). Removing the XXX now.